class.jetpack-connection-banner.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Automattic\Jetpack\Assets\Logo;
+use Automattic\Jetpack\Asset_Tools;
```

Comment:
should we go `use Automattic\Jetpack\Assets\Tools;` ?

Reply:
or just `Automattic\Jetpack\Assets`

Reply:
@gititon let's go `Automattic\Jetpack\Assets` ! seems to be the preference around the team.